web-animations-2/Overview.bs

```diff
@@ -3028,10 +3049,40 @@ with which it is associated is updated.
 
 </ins>
 
-<h3 id="the-animatable-interface">The <code>Animatable</code> interface</h3>
+The <code>Animatable</code> interface mixin {#the-animatable-interface-mixin}
+----------------------------------------
 
-<div class="methods">
+Objects that may be the target of an {{KeyframeEffect}} object implement
+the {{Animatable}} interface mixin.
+
+<pre class='idl'>
+interface mixin Animatable {
+    Animation           animate(object? keyframes,
+                                optional (unrestricted double or KeyframeAnimationOptions) options = {});
+    sequence&lt;Animation&gt; getAnimations(optional GetAnimationsOptions options = {});
+};
```

Comment:
This seems to be duplicating text from level 1 as opposed to being a delta. Is that intentional?

Reply:
Yes, I thought it would be less confusing...

Reply:
I think we should stick to being a delta if possible otherwise it's going to be really hard to know what the intentional changes are when we go to merge it 😬

Reply:
Yeah I think we only need to say that we add rangeStart and rangeEnd to KeyframeAnimationOptions here, similar to how e.g. web-animations-2 adds iterationComposite to KeyframeEffectOptions https://www.w3.org/TR/web-animations-2/#the-keyframeeffectoptions-dictionary

Reply:
Done.